funtik-scaffolding/src/main/scala/ru/primetalk/funtik/environment/ViewWorldStateT.scala

```diff
@@ -0,0 +1,30 @@
+package ru.primetalk.funtik.environment
+
+import org.scalajs.dom.CanvasRenderingContext2D
+import org.scalajs.dom.ext.Color
+import ru.primetalk.funtik.environment.geom2d.Geom2dUtils._
+
+trait ViewWorldStateT extends ViewWorldMap with EnvironmentModel {
+  implicit object ViewWorldState extends View[WorldState] {
+    implicit val displayView: ViewDisplay[Boolean] =
+      new ViewDisplay[Boolean](if(_) Color.Green else Color.apply("#70a0a0"))
+
+    override def render(t: WorldState, ctx: CanvasRenderingContext2D): Unit = {
+      t.worldPointMap.render(ctx)
+      setTransform(t.worldPointMap, ctx)
+      ctx.strokeStyle = Color.Red.toString()
+      ctx.rotate(math.Pi - t.robotEnvState.rotation)
+      ctx.lineWidth = 0.3
+      ctx.moveTo(t.robotEnvState.position.x, t.robotEnvState.position.y - 2)
```

Comment:
I think it will be better to extract `t.robotEnvState.position` to separate variable in the function